### ISSUE TITLE
Release 3.1.8 breaks language configurator

### ISSUE THREAD

**Issue by tgalery** [sweble/sweble-wikitext#74]
```
val config = LanguageConfigGenerator.generateWikiConfig("ja")
SLF4J: Failed to load class "org.slf4j.impl.StaticLoggerBinder".
SLF4J: Defaulting to no-operation (NOP) logger implementation
SLF4J: See http://www.slf4j.org/codes.html#StaticLoggerBinder for further details.
java.lang.IllegalArgumentException: No alias registered for parser function `convert'.
  at org.sweble.wikitext.engine.config.WikiConfigImpl.addParserFunction(WikiConfigImpl.java:451)
  at org.sweble.wikitext.engine.config.WikiConfigImpl.addParserFunctionGroup(WikiConfigImpl.java:432)
  at org.sweble.wikitext.engine.utils.DefaultConfig.addParserFunctions(DefaultConfig.java:584)
  at org.sweble.wikitext.engine.utils.LanguageConfigGenerator.generateWikiConfig(LanguageConfigGenerator.java:119)
  at org.sweble.wikitext.engine.utils.LanguageConfigGenerator.generateWikiConfig(LanguageConfigGenerator.java:83)
  at org.sweble.wikitext.engine.utils.LanguageConfigGenerator.generateWikiConfig(LanguageConfigGenerator.java:68)
```
  ... 42 elided

**Comment by hannesd** [comment]
Sorry for that :/
Can you check whether 7fd3dfef98945bbd2ccdd8c2976c79b6e15773c5 fixes the problem?

**Comment by tgalery** [comment]
Just tested and new dev branch fixed this. Any possibility of spinning a release so I can close this ? Or somehow updating the `3.1.8` version in maven central ?

**Comment by hannesd** [comment]
Released 3.1.9. Might take a while until it becomes visible on maven central.
